src/validation.cairo

```diff
@@ -224,7 +221,7 @@ fn fee_and_merkle_root(block: @Block) -> Result<(u64, Hash), ByteArray> {
     Result::Ok((total_fee, merkle_root(ref txids)))
 }
 
-fn validate_coinbase(block: @Block, total_fees: u64, block_height: u32) -> Result<(), ByteArray> {
+fn validate_coinbase(block: @Block, total_fees: u64, block_height: i32) -> Result<(), ByteArray> {
```

Comment:
It's ok to use `i32` in `ChainState` for now, but let's not use it elsewhere (cast early).

Reply:
I can do it with Option if it's more appropriate